frontend/package.json

```diff
@@ -5,17 +5,11 @@
   "scripts": {
     "typecheck": "tsc --noEmit",
     "test": "npm run typecheck && env TZ=UTC jest",
-    "dev": "next",
-    "dev:local-wamp-testnet": "dotenv -c -e ../testnet.env -- next",
-    "dev:local-wamp-mainnet": "dotenv -c -e ../mainnet.env -- next",
-    "dev:production-wamp-testnet": "dotenv -c -e ../testnet.env -e ../production-wamp.env -- next",
-    "dev:production-wamp-mainnet": "dotenv -c -e ../mainnet.env -e ../production-wamp.env -- next",
+    "dev": "dotenv -c -e ../frontend.env -- next",
```

Comment:
I made scripts section easier to comprehend now - with just `dev` and `dev:production-wamp`, I hope to make it even simpler with moving away from `wamp` someday :)

Reply:
Yep, thanks!